README.md

```diff
@@ -63,6 +63,17 @@ accelerate launch examples/simulacra.py
 python -m trlx.sweep --config configs/sweeps/ppo_sweep.yml examples/ppo_sentiments.py
 ```
 
+## Logging
+
+trlX uses the standard Python `logging` library to log training information to the console. The default logger is set to `INFO` level, which means that `INFO`, `WARNING`, `ERROR`, and `CRITICAL` level messages will be printed to standard output. To control the verbosity, you can set the `TRLX_LOG_LEVEL` environment variable to one of the standard logging [level names](https://docs.python.org/3/library/logging.html#logging-levels).
+
+The following example sets the log level to `WARNING` for trlX logging events.
+
+```
+export TRLX_LOG_LEVEL=WARNING
+```
+
+> 💡 Tip: To reduce the amount of logging output, you might find it helpful to change log levels of third-party libraries. For the `transformers` library, try setting `transformers.logging.set_verbosity_error()`.
```

Comment:
That's really handy, they've added some annoying logs in 4.25.1 (especially if you set prompts to `[<eos>]`). Can this method be used for trlx as well?

Reply:
Yup! I should clarify this - I meant to suggest calling `transformers.logging.set_verbosity_error()` from a user's trlx script e.g. at the top of `examples/ilql_sentiments.py`.

Reply:
(Let me know if it's any clearer 😄 )

Reply:
Yes I understood, I will also clarify that I meant to ask wether you currently do `trlx.logging.set_verbosity_error()`

Reply:
No, not currently. Maybe we should include logging methods in a similar way to transformers. What do you think?

Reply:
I mean since we're already half-way there we might as well indulge, unless it's harder than it seems of course 🙂

Reply:
😂 will do!